package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@akadenia/logger",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
with the new semantic release, we shouldn't bump the version manually anymore

Reply:
i've reverted changes made to package.json and package-lock.json